platform/osx/export/export.cpp

```diff
@@ -575,8 +576,8 @@ Error EditorExportPlatformOSX::export_project(const Ref<EditorExportPreset> &p_p
 			_fix_plist(p_preset, data, pkg_name);
 		}
 
-		if (file.begins_with("Contents/MacOS/godot_")) {
-			if (file != "Contents/MacOS/" + binary_to_use) {
+		if (file.begins_with("Contents/MacOS/godot")) {
```

Comment:
I don't understand the logic behind this check in the first place, can anyone enlighten me? Is it still necessary?

Reply:
Been there since Godot's code was freed. Try asking @reduz.